crates/llvm-context/src/polkavm/evm/call.rs

```diff
@@ -60,7 +59,7 @@ where
     let arguments = &[
         flags.as_basic_value_enum(),
         address_pointer.value.as_basic_value_enum(),
-        context.integer_const(64, 0).as_basic_value_enum(),
+        gas.as_basic_value_enum(),
```

Comment:
I'm afraid this needs to be set to 0 (effectively ignoring it) until we figure out how to handle gas limits. I got some ideas but this is a later PR.